src/deepLink/deepLink.ts

```diff
@@ -0,0 +1,101 @@
+import type { Uri } from 'vscode';
+import type { GitRemote } from '../git/models/remote';
+import type { Repository } from '../git/models/repository';
+
+export enum DeepLinkType {
+	Branch = 'b',
+	Commit = 'c',
+	Remote = 'r',
+	Tag = 't',
+}
+
+export enum DeepLinkServiceState {
```

Comment:
Make this a `const` enum unless we need reverse look ups (it didn't look like it). I would also suggest using numbers as with esbuild we will get call-site replacement. Or have no values, which will default to auto number, unless we are persisting these states and have to keep them constant across GitLens versions

Reply:
Done in commit: [Update enums](https://github.com/gitkraken/vscode-gitlens/pull/2471/commits/45a4b650d2d2b719c7ee4e4e5af9c08728b47d89)